ext/hal/nxp/mcux/drivers/imx/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2018, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+zephyr_include_directories(.)
+
+zephyr_sources_ifdef(CONFIG_GPIO_MCUX_IGPIO   fsl_gpio.c)
```

Comment:
Will drivers/imx be able to handle both imx-rt and imx6/7/8?

Reply:
That was the idea, but I found an exception today. This works for rt, imx6, imx7d, and imx8; but not imx7ulp.